app/javascript/components/patient/Patient.js

```diff
@@ -19,6 +20,7 @@ class Patient extends React.Component {
       expandArrivalNotes: false,
       expandPlannedTravelNotes: false,
       primaryLanguageDisplayName: null,
+      isMinor: props.details && props.details.date_of_birth && moment(props.details.date_of_birth, 'YYYY-MM-DD').isAfter(moment().subtract(18, 'years')),
```

Comment:
now we are doing this calculation in the front and backend, can we just add it to the patient at this point?

Reply:
I'm very leery of adding it to the patient model directly because whether the patient is a minor can change over time without any input from a user, and handling that case seems more annoying than just having a 2-line function on both frontend and backend.

Reply:
I can see the argument both ways, I guess I am just thinking about how we handle HoH but that is a totally different use case. resolving